src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -65,6 +65,10 @@
     public String userDemotedPlayer = "%prefix% &7%promoter% has demoted %player% to %rank%.";
     public String transferredOwnership = "%prefix% &7%oldowner% has transferred Island ownership to %newowner%.";
     public String islandValue = "%prefix% &b&l * &7%rank%: &b%value%";
+    public String cannotChangeName = "%prefix% &7You cannot change the island name!";
+    public String islandNameTooLong = "%prefix% %name% is too long for the island name, The maximum size is %max_length%";
+    public String islandNameTooShort = "%prefix% %name% is too short for the island name, The minimum size is %min_length%";
```

Comment:
"%prefix% %name% is too long for the island name, the minimum length is %min_length%"